src/coreclr/tools/ILVerification/ILImporter.Verify.cs

```diff
@@ -1565,7 +1568,7 @@ void ImportCall(ILOpcode opcode, int token)
                 }
             }
             else
-            if (methodType != null)
+            if (methodType != null && !isFunctionPointer)
```

Comment:
This does not look right to me. What is the `isFunctionPointer` flag trying to achieve here?

Reply:
In this case the stack at this point is empty (it was consumed above), so Pop() call was throwing.

Reply:
I think it is a bad test. See my other comment.

Reply:
You are right, fixed the test so it is not needed.